db_mutex/__init__.py

```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-from .db_mutex import db_mutex
```

Comment:
@wesleykendall looks like there was just one code example using the old import. most of our other projects were updated as well so all model imports look like ```from package.models import Model```